proctor-tomcat-deps/build.gradle

```diff
@@ -4,4 +4,20 @@ indeedLibrary.name = 'proctor-tomcat-deps'
 dependencies {
     api libs.tomcatElApi
     api libs.tomcatJasperEl
-}
\ No newline at end of file
+}
+
+task editPom(type: Exec) {
+    commandLine './editPom.sh'
```

Comment:
Just try to understand what is the file ./build/publications/maven/pom-default.xml
and why do we need to delete these 2 lines?

Reply:
Gradle build will create the pom-default.xml. Gradle publish will move this change the name and upload to nexus/maven. 

Prior to switching to gradle maven allows no version being defined in the pom.xml but gradle requires and outputs a version in the pom.xml Defining a version breaks repos which also defines the tomcat libraries in their build. This is a hack to fix it. Ideal future is getting rid of shading and investigate upgrading tomcat deps. I will DM you links to view to help make more sense.